python/paddle/distributed/auto_parallel/api.py

```diff
@@ -388,6 +392,9 @@ def reshard(dist_tensor, mesh, placements):
             dist_attr._set_partial_dims(partial_dims)
 
         return paddle.base.core.reshard(dist_tensor, dist_attr)
+    elif in_pir_mode():
+        dim_map = to_dim_map(placements, dist_tensor.ndim)
+        return paddle._pir_ops.reshard(dist_tensor, mesh, dim_map, placements)
```

Comment:
The api should take dims_mapping or placement, but not both.

Reply:
done, thanks!